kotlin-native/build-tools/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt

```diff
@@ -90,7 +90,7 @@ open class FrameworkTest : DefaultTask(), KonanTestExecutable {
 
     fun Language.filesFrom(dir: String): FileTree = project.fileTree(dir) {
         // include only files with the language extension
-        it.include("*${this.extension}")
+        include("*${this@filesFrom.extension}")
```

Comment:
Was there any changes in SAM-objects? It seems wierd that all `it`'s should be changed to `this`.

Reply:
that true, see my previous message.